index.html

```diff
@@ -1255,6 +1239,231 @@ <h3>
+            <li>If |languageMap| is not an [=ordered map=], return.
+            </li>
+            <li>Let |languageTags:ordered set| be the [=map/keys=] of
+            |languageMap:ordered map|.
+            </li>
+            <li>Set |map|[|member|] to a new [=ordered map=].
+            </li>
+            <li>[=Set/For each=] |languageTag:string| of |languageTags|, run
+            [=process a localized text value=], passing
+            |languageMap|[|languageTag|] as the [=localized value=],
+            |languageTag|, |map|, |member|, and |manifest direction|.
+            </li>
+          </ol>
+          <p>
+            To <dfn>process a localized text value</dfn>, given [=string=] or
+            [=ordered map=] |localizedValue|, [=string=] |languageTag:string|,
+            [=ordered map=] |map:ordered map|, [=string=] |member:string|, and
+            [=text-direction=] |manifest direction:string|:
+          </p>
+          <ol class="algorithm">
+            <li>Let |normalizedValue:ordered map| be an [=ordered map=].
+            </li>
+            <li>If |localizedValue| is a [=string=], [=map/set=]
+            |normalizedValue|["value"] to |localizedValue:string|.
+            </li>
+            <li>If |localizedValue:ordered map| is an [=ordered map=]:
+              <ol>
+                <li>If "value" [=map/exists=] in |localizedValue|, [=map/set=]
+                |normalizedValue|["value"] to |localizedValue|["value"].
+                </li>
+                <li>If "lang" [=map/exists=] in |localizedValue|, [=map/set=]
+                |normalizedValue|["lang"] to |localizedValue|["lang"].
+                </li>
+                <li>If "dir" [=map/exists=] in |localizedValue|, [=map/set=]
+                |normalizedValue|["dir"] to |localizedValue|["dir"].
+                </li>
+              </ol>
+            </li>
+            <li>If "value" does not [=map/exist=] in |normalizedValue|, or
+            |normalizedValue|["value"] is not a [=string=], return.
+            </li>
+            <li>If "lang" does not [=map/exist=] in |normalizedValue|, or
+            |normalizedValue|["lang"] is not a [=string=], [=map/set=]
+            |normalizedValue|["lang"] to |languageTag|.
+            </li>
+            <li>If "dir" does not [=map/exist=] in |normalizedValue|, or
+            [=text-direction list=] does not [=list/contain=]
+            |normalizedValue|["dir"], [=map/set=] |normalizedValue|["dir"] to
+            |manifest direction|.
+            </li>
+            <li>[=Map/Set=] |map|[|member|][|languageTag|] to
+            |normalizedValue|.
+            </li>
+          </ol>
+          <p class="note">
+            The [=process a localized text value=] algorithm takes both a
+            [=string=] or an [=ordered map=] for the [=localized value=]
+            parameter, but the processed result will be normalized into an
+            [=ordered map=] with the `value`, `lang`, and `dir` members set.
```

Comment:
it should be possible to link them like this:

```suggestion
            [=ordered map=] with the [=manifest/*_localized/value=], [=manifest/*_localized/lang=], and [=manifest/*_localized/dir=] members set.
```